share/spice/octopart/octopart.js

```diff
@@ -43,14 +43,19 @@
                 sourceName: 'Octopart',
                 sourceUrl : 'http://octopart.com/search?q=' + api_result.request.q
             },
+         
             normalize: function(item) {
-
-                item = item.item; // our item object is wrapped in an "item" property
-
+               
+                var item = item.item;  // our item object is wrapped in an "item" property
```

Comment:
Seeing as we're declaring multiple variables it makes sense to remove the 2nd `var`:

```js
var item = item.item, // ...
    images = ...,
    datasheet = ...;
```